src/BuildTasks/RoslynTools.BuildTasks.nuspec

```diff
@@ -5,9 +5,9 @@
   </metadata>
   <files>
     $CommonFileElements$
-    <file src="netcoreapp3.1\publish\RoslynTools.*" target="tools\netcoreapp3.1" />
-    <file src="netcoreapp3.1\publish\NuGet.Versioning.*" target="tools\netcoreapp3.1" />
-    <file src="netcoreapp3.1\publish\System.*" target="tools\netcoreapp3.1" />
+    <file src="net9.0\publish\RoslynTools.*" target="tools\net9.0" />
```

Comment:
Not ideal, I am thinking to move all nuspec files to csproj if possible. plan to do it in later PR after arcade sdks are updated